### PR TITLE
feat: add blockchain names

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx [hunk 1]
```diff
@@ -38,16 +38,16 @@ export const EVMHomeAdaptorProvider = ({
...
     switch (Number(id)) {
-      case 5:
-        return "Localhost";
       case 1:
-        return "Mainnet";
+        return "Ethereum Mainnet";
+      case 2:
+        return "Cere Mainnet (Testnet)";
       case 3:
         return "Ropsten";
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Add Ethereum in front

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx [hunk 2]
```diff
@@ -38,16 +38,16 @@ export const EVMHomeAdaptorProvider = ({
...
       case 4:
         return "Rinkeby";
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Add Ethereum in front

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx [hunk 3]
```diff
@@ -38,16 +38,16 @@ export const EVMHomeAdaptorProvider = ({
...
-      // case 5:
-      //   return "Goerli";
+      case 5:
+        return "Goerli";
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Add Ethereum in front

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx [hunk 4]
```diff
@@ -62,6 +62,10 @@ export const EVMHomeAdaptorProvider = ({
         return "CELO - Baklava Testnet";
       case 1749641142:
         return "Besu";
+      case 137:
+        return "Polygon Mainnet";
+      case 80001:
+        return "Mumbai";
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Add Polygon in front

**Review by AndreiNavoichyk — APPROVED** [review]

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx (hunk 1 above)
Comment (karthikeyan-cere) [reply src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Added

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx (hunk 2 above)
Comment (karthikeyan-cere) [reply src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Added

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx (hunk 3 above)
Comment (karthikeyan-cere) [reply src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Added

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx (hunk 4 above)
Comment (karthikeyan-cere) [reply src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Added

**Pull request merged**